### PR TITLE
Add gargish leather and studded armor

### PR DESCRIPTION
**Pull request by immaciek**
author: @ijurek

### CODE REVIEW

**Comment by kamronbatman** [comment]
With the way the CLA is set up, I need @ijurek to sign the CLA. Or you (or me or someone else) needs to redo the commits and resubmit the PR.
Sorry this is so complicated.

I can take this PR and recode it and make a new one.

**Comment by immaciek** [comment]
> With the way the CLA is set up, I need @ijurek to sign the CLA. Or you (or me or someone else that has already signed the CLA) needs to redo the commits and resubmit the PR.
> Sorry this is so complicated.
> 
> I can take this PR and recode it and make a new one.

Hey - he did sign the CLA, it wouldn't pass the checks otherwise.

File: Projects/UOContent/Items/Armor/Leather/FemaleGargishLeatherArms.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+namespace Server.Items
+{
+
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/FemaleGargishLeatherArms.cs:3]: extra line

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherArms.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+
+namespace Server.Items
+{
+    class GargishLeatherArms : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherArms.cs:4]: Add `public` modifier

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherArms.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherArms.cs:1]: Remove extra line on all files that have it. (I also wish we had a good linter to do this automatically)

**Comment by kamronbatman** [comment]
This looks great! Can you add back the TypeAliases?

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherArmsType1.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
+namespace Server.Items
+{
+    public class GargishLeatherArmsType1 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherArmsType1.cs:3]:
```suggestion
    [TypeAlias("Server.Items.GargishLeatherArms")]
    public class GargishLeatherArmsType1 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherArmsType2.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
+namespace Server.Items
+{
+    public class GargishLeatherArmsType2 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherArmsType2.cs:3]:
```suggestion
    [TypeAlias("Server.Items.FemaleGargishLeatherArms")]
    public class GargishLeatherArmsType2 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherChestType1.cs [hunk 6]
```diff
@@ -0,0 +1,40 @@
+namespace Server.Items
+{
+    public class GargishLeatherChestType1 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherChestType1.cs:3]:
```suggestion
    [TypeAlias("Server.Items.GargishLeatherChest")]
    public class GargishLeatherChestType1 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherChestType2.cs [hunk 7]
```diff
@@ -0,0 +1,42 @@
+namespace Server.Items
+{
+    public class GargishLeatherChestType2 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherChestType2.cs:3]:
```suggestion
    [TypeAlias("Server.Items.FemaleGargishLeatherChest")]
    public class GargishLeatherChestType2 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherKiltType1.cs [hunk 8]
```diff
@@ -0,0 +1,40 @@
+namespace Server.Items
+{
+    public class GargishLeatherKiltType1 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherKiltType1.cs:3]:
```suggestion
    [TypeAlias("Server.Items.GargishLeatherKilt")]
    public class GargishLeatherKiltType1 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherKiltType2.cs [hunk 9]
```diff
@@ -0,0 +1,41 @@
+namespace Server.Items
+{
+    public class GargishLeatherKiltType2 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherKiltType2.cs:3]:
```suggestion
    [TypeAlias("Server.Items.FemaleGargishLeatherKilt")]
    public class GargishLeatherKiltType2 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherLegsType1.cs [hunk 10]
```diff
@@ -0,0 +1,41 @@
+namespace Server.Items
+{
+    public class GargishLeatherLegsType1 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherLegsType1.cs:3]:
```suggestion
    [TypeAlias("Server.Items.GargishLeatherLegs")]
    public class GargishLeatherLegsType1 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Leather/GargishLeatherLegsType2.cs [hunk 11]
```diff
@@ -0,0 +1,41 @@
+namespace Server.Items
+{
+    public class GargishLeatherLegsType2 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Leather/GargishLeatherLegsType2.cs:3]:
```suggestion
    [TypeAlias("Server.Items.FemaleGargishLeatherLegs")]
    public class GargishLeatherLegsType2 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Studded/GargishStuddedArmsType1.cs [hunk 12]
```diff
@@ -0,0 +1,40 @@
+namespace Server.Items
+{
+    public class GargishStuddedArmsType1 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Studded/GargishStuddedArmsType1.cs:3]:
```suggestion
    [TypeAlias("Server.Items.GargishStuddedArms")]
    public class GargishStuddedArmsType1 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Studded/GargishStuddedArmsType2.cs [hunk 13]
```diff
@@ -0,0 +1,40 @@
+namespace Server.Items
+{
+    public class GargishStuddedArmsType2 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Studded/GargishStuddedArmsType2.cs:3]:
```suggestion
    [TypeAlias("Server.Items.FemaleGargishStuddedArms")]
    public class GargishStuddedArmsType2 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Studded/GargishStuddedChestType1.cs [hunk 14]
```diff
@@ -0,0 +1,42 @@
+namespace Server.Items
+{
+    public class GargishStuddedChestType1 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Studded/GargishStuddedChestType1.cs:3]:
```suggestion
    [TypeAlias("Server.Items.GargishStuddedChest")]
    public class GargishStuddedChestType1 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Studded/GargishStuddedChestType2.cs [hunk 15]
```diff
@@ -0,0 +1,41 @@
+namespace Server.Items
+{
+    public class GargishStuddedChestType2 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Studded/GargishStuddedChestType2.cs:3]:
```suggestion
    [TypeAlias("Server.Items.FemaleGargishStuddedChest")]
    public class GargishStuddedChestType2 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Studded/GargishStuddedKiltType1.cs [hunk 16]
```diff
@@ -0,0 +1,43 @@
+namespace Server.Items
+{
+    public class GargishStuddedKiltType1 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Studded/GargishStuddedKiltType1.cs:3]:
```suggestion
    [TypeAlias("Server.Items.GargishStuddedKilt")]
    public class GargishStuddedKiltType1 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Studded/GargishStuddedKiltType2.cs [hunk 17]
```diff
@@ -0,0 +1,41 @@
+namespace Server.Items
+{
+    public class GargishStuddedKiltType2 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Studded/GargishStuddedKiltType2.cs:3]:
```suggestion
    [TypeAlias("Server.Items.FemaleGargishStuddedKilt")]
    public class GargishStuddedKiltType2 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Studded/GargishStuddedLegsType1.cs [hunk 18]
```diff
@@ -0,0 +1,42 @@
+namespace Server.Items
+{
+    public class GargishStuddedLegsType1 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Studded/GargishStuddedLegsType1.cs:3]:
```suggestion
    [TypeAlias("Server.Items.GargishStuddedLegs")]
    public class GargishStuddedLegsType1 : BaseArmor
```

File: Projects/UOContent/Items/Armor/Studded/GargishStuddedLegsType2.cs [hunk 19]
```diff
@@ -0,0 +1,42 @@
+namespace Server.Items
+{
+    public class GargishStuddedLegsType2 : BaseArmor
```
Comment (kamronbatman) [inline Projects/UOContent/Items/Armor/Studded/GargishStuddedLegsType2.cs:3]:
```suggestion
    [TypeAlias("Server.Items.FemaleGargishStuddedLegs")]
    public class GargishStuddedLegsType2 : BaseArmor
```

**Review by kamronbatman — CHANGES_REQUESTED** [review]

**Review by kamronbatman — APPROVED** [review]

**Pull request merged**